src/setuptools_scm/version.py

```diff
@@ -139,16 +173,18 @@ def format_next_version(self, guess_next, fmt="{guessed}.dev{distance}", **kw):
         return self.format_with(fmt, guessed=guessed)
 
 
-def _parse_tag(tag, preformatted):
+def _parse_tag(tag, preformatted, config):
     if preformatted:
         return tag
     if VERSION_CLASS is None or not isinstance(tag, VERSION_CLASS):
-        tag = tag_to_version(tag)
+        tag = tag_to_version(tag, config)
     return tag
 
 
-def meta(tag, distance=None, dirty=False, node=None, preformatted=False, **kw):
-    parsed_version = _parse_tag(tag, preformatted)
+def meta(tag, config=None, distance=None, dirty=False, node=None, preformatted=False, **kw):
```

Comment:
unfortunately this is a breaking api change, please move config to the end right after preformatted - this one really makes me want to start to use keyword only args from python3 sonner

Reply:
ok